app/models/passenger.rb

```diff
@@ -7,6 +7,9 @@ class Passenger < ApplicationRecord
                     format: { with: VALID_EMAIL_REGEX }, uniqueness: true
   STATUSES = %w[Alumni Faculty Staff Student].freeze
   validates :status, inclusion: { in: STATUSES, allow_blank: true }
+  validates :spire,
```

Comment:
Why do we need this?

Reply:
Not just this validation, but SPIRE for passengers in general.

Reply:
I think that was one of the things the ST specifically asked to have in the database. Not sure what the rationale was, though.

Reply:
OK.